include/api/syscall.h

```diff
@@ -16,19 +16,6 @@
 
 #define TIME_ARG_SIZE (sizeof(ticks_t) / sizeof(word_t))
 
-#ifdef CONFIG_KERNEL_MCS
-#define MCS_DO_IF_BUDGET(_block) \
-    updateTimestamp(); \
-    if (likely(checkBudgetRestart())) { \
-        _block \
-    }
-#else
-#define MCS_DO_IF_BUDGET(_block) \
-    { \
```

Comment:
This was done this way because the if(1) statements by using a ternary approach required proof rework.

Reply:
Ok, that explans it. I will add this as a comment then instead.